static/app/utils/analytics/replayAnalyticsEvents.tsx

```diff
@@ -134,6 +144,10 @@ export const replayEventMap: Record<ReplayEventKey, string | null> = {
   'replay.details-tab-changed': 'Changed Replay Details Tab',
   'replay.details-time-spent': 'Time Spent Viewing Replay Details',
   'replay.frame-after-background': 'Replay Frame Following Background Frame',
+  'replay.hydration-error.issue-details-opened': 'Hydration Issue Details Opened',
+  'replay.hydration-modal.opened': 'Hydration Modal Opened',
```

Comment:
This is duplicated from the button prop at https://github.com/getsentry/sentry/blob/a997487d02475a32b0200842e84def606dde9fd8/static/app/components/replays/breadcrumbs/openReplayComparisonButton.tsx#L39. Not sure what overrides what here. Or if the button's eventName can be removed after adding this definition.

Reply:
i don't think you need to add this one to this file (`replay.hydration-modal.opened`), since it's already defined with the button analytics. and you mentioned it's already being sent to amplitude